src/cc/arduino/plugins/wifi101/flashers/java/SSLCertDownloader.java

```diff
@@ -82,7 +82,7 @@ public void checkServerTrusted(X509Certificate[] xcs, String string) throws Cert
 		connection.setSSLSocketFactory(ssl.getSocketFactory());
 
 		Certificate[] certificates = null;
-		if (connection.getResponseCode() == 200) {
+		if (connection.getResponseCode() > 0) {
```

Comment:
Either way, I'm fine with the change, you can merge it with or without the check